libs/application/templates/financial-aid/src/fields/Status/ApprovedAlert/ApprovedAlert.tsx

```diff
@@ -21,7 +21,7 @@ const ApprovedAlert = ({ events }: Props) => {
       return events.find((el) => el.eventType === ApplicationEventType.APPROVED)
         ?.comment
     }
-  }, [events])
+  }, [events])?.split('\n')[1]
```

Comment:
why only show part of the approved comment?

Reply:
I want to challenge this, I think we should just show the full comment, not all employees are going to remember to put a new line after approving, I think this can easily wind up showing the wrong comment or no comment. And also what if they put two new lines, then you will only be showing the one? 

If they really do insist on removing the first line, than I rather see that the you split the array and remove the first element and show the rest

Reply:
Nice, well spotted! I forgot to think about the case where the user enters multiple lines 🤕